research/frontend/css/index.css

```diff
@@ -0,0 +1,55 @@
+html, body {
+    width: 100%;
+    margin: 0 auto;
+    background-color: #f7f7f7;
+    color: black;
+  }
```

Comment:
Take care of your code format. No spaces required here.

Reply:
okay, I'll check it